src/UserALEWebExtension/optionsPage.html

```diff
@@ -50,7 +50,39 @@ <h1>Options</h1>
       <br />
 
       <div align="right">
-        <button type="submit" id="submit">Save</button>
+        <button type="submit" id="submitOptions">Save Changes</button>
+      </div>
+    </form>
+    <h1>Report an Issue</h1>
+    <form name="issueForm" id="issueForm">
+      <label>Issue Type</label><br />
+      <label>
+        <input
+          type="radio"
+          id="bug"
+          name="issueType"
+          value="Bug"
+          checked="checked"
+        />
+        Bug </label
+      ><br />
+
+      <label>
+        <input type="radio" id="feat" name="issueType" value="Feature" />
+        Feature Request </label
+      ><br />
+      <br />
```

Comment:
The ol' `<br/>` to serve as padding instead of proper styles ;). Will let it slide for now.